lib/puppet/provider/zabbix_host/ruby.rb

```diff
@@ -18,6 +18,7 @@ def create
     zabbix_user = @resource[:zabbix_user]
     zabbix_pass = @resource[:zabbix_pass]
     apache_use_ssl = @resource[:apache_use_ssl]
+	enabled = @resource[:enabled]
```

Comment:
please check the format and align this varible with the ones above. Please don't use hard tabs